src/runtime/data/kata-collect-data.sh

```diff
@@ -250,7 +250,6 @@ show_runtime_configs()
 	# add in the standard defaults for good measure "just in case"
 	configs+=" /etc/kata-containers/configuration.toml"
 	configs+=" /usr/share/defaults/kata-containers/configuration.toml"
-	configs+=" /etc/kata-containers/configuration.toml"
```

Comment:
I think this change is not related. As well as the changes in `tools/packaging/kernel/configs/fragments/x86_64/.config`

Reply:
the config file is definitely not related, but I removed that line because it is redundant of line 250 above, if I understand the code correctly.